pyproject.toml

```diff
@@ -61,10 +61,11 @@ tests = ["pytest", "safetensors"]
 quality = ["black", "ruff", "isort"]
 # Jetstream/Pytorch support is experimental for now, it needs to be installed manually.
 # Pallas is pulled because it will install a compatible version of jax[tpu].
-jetstream-pt = [
```

Comment:
you do not need to comment this: you will only install it if you do `pip install optimum-tpu[pallas]`, otherwise it should not pull the dependency

Reply:
Ok